powerlevel9k.zsh-theme

```diff
@@ -1628,11 +1628,21 @@ prompt_virtualenv() {
 }
 
 ################################################################
-# pyenv: current active python version (with restrictions)
+# Segment to display pyenv information
 # https://github.com/pyenv/pyenv#choosing-the-python-version
+set_default POWERLEVEL9K_PYENV_PROMPT_ALWAYS_SHOW false
 prompt_pyenv() {
-  if [[ -n "$PYENV_VERSION" ]]; then
-    "$1_prompt_segment" "$0" "$2" "blue" "$DEFAULT_COLOR" "$PYENV_VERSION" 'PYTHON_ICON'
+  if ! [ -x "$(command -v pyenv)" ]; then
+    if [[ -n "$PYENV_VERSION" ]]; then
+      "$1_prompt_segment" "$0" "$2" "blue" "$DEFAULT_COLOR" "$PYENV_VERSION" 'PYTHON_ICON'
+    else
+      local pyenv_version_name="$(pyenv version-name)"
+      local pyenv_global="$(pyenv version-file-read $(pyenv root)/version)"
+      if [[ $pyenv_version_name == $pyenv_global && "$POWERLEVEL9K_PYENV_PROMPT_ALWAYS_SHOW" = false ]]; then
```

Comment:
If we flip that condition, we could save that return. That would be probably more readable, no?